pkg/platform/aws/aws.go

```diff
@@ -410,3 +412,22 @@ func (c *config) checkWorkerPoolNamesUnique() hcl.Diagnostics {
 
 	return diagnostics
 }
+
+func (c *config) checkCPUManagerPolicy() hcl.Diagnostics {
+	var diagnostics hcl.Diagnostics
+
+	for _, w := range c.WorkerPools {
+		switch w.CPUManagerPolicy {
+		case "", "none", "static":
+			continue
+		default:
+			diagnostics = append(diagnostics, &hcl.Diagnostic{
+				Severity: hcl.DiagError,
+				Summary:  "invalid cpu_manager_policy",
+				Detail:   fmt.Sprintf("Worker pool '%s' has invalid cpu_manager_policy '%s'", w.Name, w.CPUManagerPolicy),
```

Comment:
`%q` should be used for quoting.
```suggestion
				Detail:   fmt.Sprintf("Worker pool %q has invalid cpu_manager_policy %q", w.Name, w.CPUManagerPolicy),
```